misc/MajorityVoting.ipynb

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
<p>This should not be a separate notebook? Just a side uitl which can be used with all notebook/model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>